usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,117 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+@pytest.mark.parametrize("cluster_name", ["ClusterName"])
+def test_cluster_create_valid(
+        valid_session_credentials,
+        valid_nodes,
+        network_interface,
+        cluster_name):
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    """@pylatest api/gluster.cluster_import
+        .. test_step:: 1
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 1
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    network = None
+    node_ids = []
+    first = True
+    for x in valid_nodes:
+        if "tendrl/server" not in x["tags"]:
+            if first:
```

Comment:
It would write this down a bit different way:

```
for x in valid_nodes:
    if "tendrl/server" in x["tags"]:
        continue
    if first:
        ...
    ips = ...
    ...
```

This way, you limit nesting of the code, which makes it easier to read and reason about.